homeassistant/components/dexcom/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup Dexcom integration",
+        "description": "Enter Dexcom Share credentials, note that Dexcom Share must be enabled on your Dexcom app.",
+        "data": {
+          "username": "Username",
```

Comment:
Please use as many common strings as possible.

Here are the common strings for the config flow:
https://github.com/home-assistant/core/blob/4170eb0f83ee5fef11eef1b656d3aa4696c67b95/homeassistant/strings.json#L20-L55

Here's an example:
https://github.com/home-assistant/core/blob/4170eb0f83ee5fef11eef1b656d3aa4696c67b95/homeassistant/components/adguard/strings.json#L8